ini.go

```diff
@@ -123,6 +123,9 @@ type LoadOptions struct {
 	ReaderBufferSize int
 	// AllowNonUniqueSections indicates whether to allow sections with the same name multiple times.
 	AllowNonUniqueSections bool
+	// AllowShortCircuit allows to ignore other configuration sources after loading the first available configuration source.
+	// Notice: if AllowShortCircuit and Loose are both true, AllowShortCircuit has priority over Loose.
+	AllowShortCircuit bool
```

Comment:
I think for convention, these new lines should be inserted after line 83, without prefix `Allow` because it is only used at "load" time not for key and/or values.

Reply:
All right, I got it. I'll submit it together later.